site/content/docs/5.1/forms/layout.md

```diff
@@ -12,7 +12,6 @@ Every group of form fields should reside in a `<form>` element. Boosted provides
 
 - New to browser forms? Consider reviewing [the MDN form docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form) for an overview and complete list of available attributes.
 - `<button>`s within a `<form>` default to `type="submit"`, so strive to be specific and always include a `type`.
-- You can disable every form element within a form with the `disabled` attribute on the `<form>`.
```

Comment:
Why are we removing this ?

Reply:
Adding the `disabled` attribute to a `<form>` does nothing. It is probably a confusion with the `<fieldset>` which allows you to have this behavior. Since this section is only about `<form>, this sentence has been removed in Bootstrap.